bigtable-dataflow-parent/bigtable-beam-import/README.md

```diff
@@ -1,44 +1,220 @@
 # HBase Sequence Files to Cloud Bigtable using Beam
 
-This project supports importing and exporting HBase Sequence Files to Google Cloud Bigtable using
-Cloud Dataflow.
+This folder contains tools to support importing and exporting HBase data to
+Google Cloud Bigtable using Cloud Dataflow.
 
-## Instructions
-[//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
-Download [the import/export jar](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-beam-import/1.14.1/bigtable-beam-import-1.14.1-shaded.jar), which is an aggregation of all required jars.
+## Setup 
 
-Please pay attention to the Cluster CPU usage and adjust the number of Dataflow workers accordingly.
+To use the tools in this folder, you can download them from the maven repository, or
+you can build them using Maven. 
+
+
+[//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
+### Download the jars
+Download [the import/export jars](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-beam-import/1.14.1/bigtable-beam-import-1.14.1-shaded.jar), which is an aggregation of all required jars.
 
-## Export
+### Build the jars yourself
 
-On the command line:
+Go to the top level directory and build the repo
+then return to this sub directory.
 
 ```
-java -jar bigtable-beam-import-1.14.1-shaded.jar export \
-    --runner=dataflow \
-    --project=[your_project_id] \
-    --bigtableInstanceId=[your_instance_id] \
-    --bigtableTableId=[your_table_id] \
-    --destinationPath=gs://[bucket_name]/[export_directory]/ \
-    --tempLocation=gs://[bucket_name]/[temp_work_directory]/ \
-    --maxNumWorkers=[10x number of nodes] \
+cd ../../
+mvn clean install -DskipTests=true
+cd bigtable-dataflow-parent/bigtable-beam-import
 ```
 
-## Import
+***
+#Tools
 
-Create the table in your cluster.
+## Data export pipeline
+
+You can export data into a snapshot or into sequence files. If you're migrating
+your data from HBase to Bigtable using snapshots is the preferred method. 
+
+### Exporting snapshots from HBase
+
+Perform these steps from Unix shell on HBase master node.
```

Comment:
users can launch the tool from an `HBase edge node` as they wouldn't be permitted to run processes on master nodes